### PR TITLE
feat: rebuild when package.json dependencies changed

### PR DESCRIPTION
**Pull request by await-ovo**
Fix: #600

### CODE REVIEW

File: src/load.ts [hunk 1]
```diff
@@ -94,3 +97,8 @@ export async function getDeps(cwd: string) {
 
   return deps
 }
+
+export async function getDepsHash(cwd: string) {
+  const deps = await getDeps(cwd, true)
+  return hash(deps.join(','))
```
Comment (egoist) [inline src/load.ts:103]: why not directly compare `deps.join(',')`?

Comment (egoist) [reply src/load.ts:103]: I think we also need to rebuild when the dependency version changes

Comment (egoist) [reply src/load.ts:103]: Looks good

Comment (await-ovo) [reply src/load.ts:103]: I'll add it later~

**Pull request merged**